app/assets/stylesheets/index.scss

```diff
@@ -1,13 +1,19 @@
- .title {
+.title {
   font-size: 20px;
   margin-bottom: 0;
 }
 
-.piece {
+.peace {
   padding-left: 10px;
   padding-top: 20px;
   padding-bottom: 0;
-  margin-left: 10px;
+  margin-bottom: 10px;
+  margin-left: 20px;
+  margin-right: 20px;
+  background-color: $light_blue;
+  &:hover{
+  background-color: $lighter_blue;
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Name of variable `lighter_blue` should be written in all lowercase letters with hyphens instead of underscores